src/multi/uvarint.cpp

```diff
@@ -11,53 +11,39 @@ namespace libp2p::multi {
   using common::hex_upper;
 
   UVarint::UVarint(uint64_t number) {
```

Comment:
Here in the last, 7th iteration, we leave the last 8 bits from number.
When `number = std::numeric_limits<uint64_t>::max()`, condition on line 20 will never be fulfilled, and we will successfully create instance with null `bytes_`, which will cause returning -1 value in the `UVarint::toUInt64()`
Example:
```
UVarint var{ 2^64 - 1000 };
uint64_t decoded = var.toUInt64();
```
Here `decoded` will have value 2^64 - 1, because condition on line 20 was not fulfilled and `bytes_` was resized to `0`. Next, in the `UVarint::toUInt64()` was returned value `-1`, which represented ad `2^64-1` for `uint64_t`